bots/pr/src/main/java/org/openjdk/skara/bots/pr/PullRequestCheckIssueVisitor.java

```diff
@@ -252,37 +259,45 @@ public void visit(MessageWhitespaceIssue issue) {
         } else {
             desc = "an unknown kind of whitespace (" + issue.kind().name() + ")";
         }
-        addFailureMessage(issue.check(), "The commit message contains " + desc + " on line " + issue.line());
-        readyForReview = false;
+        addMessage(issue.check(), "The commit message contains " + desc + " on line " + issue.line(), issue.severity(), false);
     }
 
     @Override
     public void visit(IssuesIssue issue) {
-        addFailureMessage(issue.check(), "The commit message does not reference any issue. To add an issue reference to this PR, " +
-                "edit the title to be of the format `issue number`: `message`.");
-        readyForReview = false;
+        addMessage(issue.check(), "The commit message does not reference any issue. To add an issue reference to this PR, " +
+                "edit the title to be of the format `issue number`: `message`.", issue.severity(), false);
     }
 
     @Override
     public void visit(ExecutableIssue issue) {
-        addFailureMessage(issue.check(), String.format("Executable files are not allowed (file: %s)", issue.path()));
-        readyForReview = false;
+        addMessage(issue.check(), String.format("Executable files are not allowed (file: %s)", issue.path()), issue.severity(), false);
     }
 
     @Override
     public void visit(SymlinkIssue issue) {
-        addFailureMessage(issue.check(), String.format("Symbolic links are not allowed (file: %s)", issue.path()));
-        readyForReview = false;
+        addMessage(issue.check(), String.format("Symbolic links are not allowed (file: %s)", issue.path()), issue.severity(), false);
     }
 
     @Override
     public void visit(BinaryIssue issue) {
-        addFailureMessage(issue.check(), String.format("Binary files are not allowed (file: %s)", issue.path()));
-        readyForReview = false;
+        addMessage(issue.check(), String.format("Binary files are not allowed (file: %s)", issue.path()), issue.severity(), false);
     }
 
     @Override
     public void visit(ProblemListsIssue issue) {
-        addFailureMessage(issue.check(), issue.issue() + " is used in problem lists: " + issue.files());
+        addMessage(issue.check(), issue.issue() + " is used in problem lists: " + issue.files(), issue.severity(), true);
+    }
+
+    @Override
+    public void visit(IssuesTitleIssue issue) {
+        List<String> messages = new ArrayList<>();
+        if (!issue.issuesWithTrailingPeriod().isEmpty()) {
+            messages.add("Found trailing period in " + String.join(" ,", issue.issuesWithTrailingPeriod()));
```

Comment:
Shouldn't the join be comma followed with space? Maybe specify that this is in the issue title?
```suggestion
            messages.add("Found trailing period in issue title for " + String.join(", ", issue.issuesWithTrailingPeriod()));
```